core/src/main/java/nl/nn/adapterframework/webcontrol/api/TransactionalStorage.java

```diff
@@ -445,17 +446,23 @@ public Response browsePipeMessages(
 		return Response.status(Response.Status.OK).entity(getMessages(storage, filter)).build();
 	}
 
-	private String[] getMessages(MultipartFormDataInput input) {
-		try {
-			Map<String, List<InputPart>> inputDataMap = input.getFormDataMap();
-			if(inputDataMap.get("messageIds") != null) {
-				String messageIds = inputDataMap.get("messageIds").get(0).getBodyAsString();
-				return messageIds.split(",");
+	private Optional<String> getAttributeValue(String name, List<Attachment> attachmentList) {
```

Comment:
try to not duplicate code, this is already in the base class?

Reply:
to get an attribute from the new Multipart object is not the same way than "multipart" from "restEasy".